.github/workflows/testx.yml

```diff
@@ -26,3 +21,8 @@ jobs:
             echo "Coverage threshold not met."
             exit 1
           fi
+
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v3
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
is it somewhere this variable?

Reply:
Good question - yes, it's an environment variable, I've set it in Settings